rules/linux/defense_evasion_selinux_configuration_creation_modification.toml

```diff
@@ -0,0 +1,79 @@
+[metadata]
+creation_date = "2024/08/28"
+integration = ["endpoint"]
+maturity = "production"
+updated_date = "2024/08/28"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects the creation or modification of the SELinux configuration file. SELinux is a security module that
+provides access control security policies. Modifications to the SELinux configuration file may indicate an attempt to
+impair defenses by disabling or modifying security tools.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.file*", "endgame-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "SELinux Configuration Creation or Modification"
```

Comment:
IIRC we don't log file overwrite, to avoid assumptions better be specific on Creation or Rename in both rule.name and description.

Reply:
You're correct. Changed it, thanks